widgetry/src/widgets/fan_chart.rs

```diff
@@ -4,10 +4,10 @@ use geom::{
     Angle, Distance, Duration, HgramValue, Histogram, PolyLine, Polygon, Pt2D, Statistic, Time,
 };
 
-use crate::widgets::line_plot::{make_legend, thick_lineseries, Yvalue};
+use crate::widgets::plots::{make_legend, thick_lineseries, Axis, PlotOptions};
 use crate::{
-    Color, Drawable, EventCtx, GeomBatch, GfxCtx, PlotOptions, ScreenDims, ScreenPt, Series, Text,
-    TextExt, Widget, WidgetImpl, WidgetOutput,
+    Color, Drawable, EventCtx, GeomBatch, GfxCtx, ScreenDims, ScreenPt, Series, Text, TextExt,
+    Widget, WidgetImpl, WidgetOutput,
 };
 
 // The X is always time
```

Comment:
We can generalize this and scatter plot in the future too. There's loads of duplicate logic in these two that also needs to get cleaned up, like determining the scale for the axis and drawing the ticks.